src/Contants/Utils.tsx

```diff
@@ -344,3 +353,39 @@ export const handleFacebookLogin = async (
       console.error('error', e);
     });
 };
+
+export const onLogout = async (user: any, dispatch: any, navigation: any) => {
+  try {
+    console.log('yes calling.........');

```

Comment:
remove log and also remove await AsyncStorage.clear(); you are not using AsyncStorage.